data/Sri Guru Granth Sahib Ji/0247.json

```
@@ -14,7 +14,7 @@
         "pronunciation_information": "",
         "type": "Sirlekh",
         "gurmukhi": {
-          "SGPC": "rwgu gauVI CMq mhlw 5 ]"
+          "SGPC": "rwgu gauVI₁ CMq mhlw 5 ]"
```

```suggestion
          "SGPC": "rwgu g₁auVI CMq mhlw 5 ]"
```

Also note unicode displays this too far to the left